### PR TITLE
[MISC] Update bundle and disable tests

### PR DESCRIPTION
**Pull request by dragomirp**
Update the bundle and mark the current tests as unstable.

### CODE REVIEW

**Review by marceloneppel — APPROVED** [review]

File: releases/latest/postgresql-k8s-bundle.yaml [hunk 1]
```diff
@@ -3,31 +3,61 @@ applications:
...
+    constraints: arch=amd64
+    revision: 26
+    scale: 1
+  s3-integrator:
+    channel: latest/edge
+    charm: s3-integrator
+    constraints: arch=amd64
+    revision: 17
+    scale: 1
+  grafana-agent-k8s:
```
Comment (taurus-forever) [inline releases/latest/postgresql-k8s-bundle.yaml:36]: JFMI, can we use the same grafana-agent-k8s for both pb and pgb on K8s?

**Review by taurus-forever — APPROVED** [review]
Tnx!

File: releases/latest/postgresql-k8s-bundle.yaml (hunk 1 above)
Comment (dragomirp) [reply releases/latest/postgresql-k8s-bundle.yaml:36]: I think so, but I haven't double checked.

**Comment by dragomirp** [comment]
Merging since this is the bundle I've published manually and to allow publishing. We can add more charms to the bundle in follow up PRs

**Pull request merged**